content/zh-cn/docs/concepts/overview/components.md

```diff
@@ -24,12 +24,10 @@ card:
 
 <!--
 When you deploy Kubernetes, you get a cluster.
-{{< glossary_definition term_id="cluster" length="all" prepend="A Kubernetes cluster consists of">}}
```

Comment:
I'd suggest just add a space here to prevent shortcode resolve.

```diff
-{{< glossary_definition term_id="cluster" length="all" prepend="A Kubernetes cluster consists of">}}
+{{ < glossary_definition term_id="cluster" length="all" prepend="A Kubernetes cluster consists of">}}
```

Reply:
OK, it has been modified